### PR TITLE
add method to set selection

### PR DESCRIPTION
**Pull request by clemos**

### CODE REVIEW

File: src/rsz/wysiwyg/wyswyg.js [hunk 1]
```diff
@@ -440,5 +440,14 @@ class Wysiwyg {
   getSelected() {
     return this.rszSelection.getSelected(this.document);
   }
+  
+  /**
+   * handle selection
+   * @param {Element} node
+   * @export
+   */
+  setSelected( node ) {
```
Comment (lexoyo) [inline src/rsz/wysiwyg/wyswyg.js]: you probably want to call ```RszSelection::unSelectAll``` here?

Comment (clemos) [inline src/rsz/wysiwyg/wyswyg.js]: I have no idea

Comment (clemos) [inline src/rsz/wysiwyg/wyswyg.js]: ;)

Comment (lexoyo) [inline src/rsz/wysiwyg/wyswyg.js]: Yes I tell you, you should :)

Comment (clemos) [inline src/rsz/wysiwyg/wyswyg.js]: done

**Pull request merged**